packages/@lwc/ssr-compiler/src/compile-template/transformers/text.ts

```diff
@@ -21,20 +24,15 @@ import type { Transformer } from '../types';
 
 const bBufferTextContent = esTemplateWithYield`
     didBufferTextContent = true;
```

Comment:
🤔 Could we make the buffer state part of the context object that we pass around? Might help address this repetition.

Reply:
Can't be fully at compile time, because directives like `lwc:if` are unknown until runtime. But maybe it could mitigate it?